aws/rust-runtime/aws-http/external-types.toml

```diff
@@ -1,11 +1,3 @@
 allowed_external_types = [
-    "aws_smithy_runtime_api::http::headers::Headers",
```

Comment:
is this actually true now? I thought the crate was empty

Reply:
The crate re-exports the types that were moved into aws-runtime.